src/components/SharedComponents/Button/LoadingButton/LoadingButton.tsx

```diff
@@ -21,11 +21,14 @@ type LoadingButtonProps = {
   disabled: boolean;
   success?: boolean;
   ariaLabel?: string;
-  onClick: MouseEventHandler<HTMLButtonElement | HTMLAnchorElement>;
+  onClick?: MouseEventHandler<HTMLButtonElement | HTMLAnchorElement>;
 } & (
   | {
       type: "button";
     }
+  | {
```

Comment:
It might be worth using the oak-components loading button instead of this one

Reply:
Ahhh missed that one in oak-components, for my reference see <https://components.thenational.academy/?path=/docs/components-molecules-oakprimarybutton--docs>

Reply:
I'm now using `<OakPrimaryButton />` and I've reverted changes to `<LoadingButton/>`